### ISSUE TITLE
1.17 "Pre-release"

### ISSUE THREAD

**Issue by ZaneJulienMOI** [KaptainWutax/SeedCracker#213]
Hey man,
i know that it is not finished for 1.17 yet, but it seems like it would work for 1.17 when building it with gradle. The thing is that i can't get that to work, so could you just build the current stage and put it into the releases as some kind of Pre-Release? Would be really nice.

**Comment by CoffeeMakesSense** [comment]
I agree with this. The dev is busy though, so don't get your hopes up.

**Comment by joogle12** [comment]
> Hey man,
> i know that it is not finished for 1.17 yet, but it seems like it would work for 1.17 when building it with gradle. The thing is that i can't get that to work, so could you just build the current stage and put it into the releases as some kind of Pre-Release? Would be really nice.

I made one. https://github.com/joogle12/SeedCracker/releases/tag/1.17-release

**Comment by listenthelisten** [comment]
> > Hey man,
> > i know that it is not finished for 1.17 yet, but it seems like it would work for 1.17 when building it with gradle. The thing is that i can't get that to work, so could you just build the current stage and put it into the releases as some kind of Pre-Release? Would be really nice.
> 
> I made one. https://github.com/joogle12/SeedCracker/releases/tag/1.17-release

In fact, the seedcracker you created may only work with the original 1.17 version.  It may not work properly if the server adds cave and cliff packets. I wonder if you can update a version for cave and cliff packets？I'd really appreciate it if you could.

**Comment by joogle12** [comment]
> > > Hey man,
> > > i know that it is not finished for 1.17 yet, but it seems like it would work for 1.17 when building it with gradle. The thing is that i can't get that to work, so could you just build the current stage and put it into the releases as some kind of Pre-Release? Would be really nice.
> > 
> > 
> > I made one. https://github.com/joogle12/SeedCracker/releases/tag/1.17-release
> 
> In fact, the seedcracker you created may only work with the original 1.17 version. It may not work properly if the server adds cave and cliff packets. I wonder if you can update a version for cave and cliff packets？I'd really appreciate it if you could.

isn't caves and cliff coming out in 1.18?

**Comment by listenthelisten** [comment]
> > > > 嘿，伙计，我知道它还没有完成1.17，但它似乎会工作1.17时，建立它与gradle。问题是，我不能让它工作，所以你可以只是建立当前阶段，并将其作为某种预发布版本？会很好的
> > > 
> > > 
> > > 我做了一个https://github.com/joogle12/SeedCracker/releases/tag/1.17-release
> > 
> > 
> > 事实上，您创建的种子爆竹可能仅适用于原始 1.17 版本。如果服务器添加洞穴和悬崖包，它可能无法正常工作。我不知道你是否可以更新一个版本的洞穴和悬崖包？
> 
> 洞穴和悬崖不是在1.18年出来的吗？地下生物群系等？

Mojang opens up caves and cliff packets for players to experience 1.18 content in advance. You can search the wiki for more information. It's named Cave & Cliffs Preview Datapack. This datapack is added to the server I am playing. My friends and I were unable to determine the seed using your tool, so we suspected whether it was because the server added this packet.

**Comment by joogle12** [comment]
> > > > > 嘿，伙计，我知道它还没有完成1.17，但它似乎会工作1.17时，建立它与gradle。问题是，我不能让它工作，所以你可以只是建立当前阶段，并将其作为某种预发布版本？会很好的
> > > > 
> > > > 
> > > > 我做了一个https://github.com/joogle12/SeedCracker/releases/tag/1.17-release
> > > 
> > > 
> > > 事实上，您创建的种子爆竹可能仅适用于原始 1.17 版本。如果服务器添加洞穴和悬崖包，它可能无法正常工作。我不知道你是否可以更新一个版本的洞穴和悬崖包？
> > 
> > 
> > 洞穴和悬崖不是在1.18年出来的吗？地下生物群系等？
> 
> Mojang opens up caves and cliff packets for players to experience 1.18 content in advance. You can search the wiki for more information. It's named Cave & Cliffs Preview Datapack. This datapack is added to the server I am playing. My friends and I were unable to determine the seed using your tool, so we suspected whether it was because the server added this packet.

hey. sorry, I don't know how to do that. I only built the mod from source and shared it.

**Comment by listenthelisten** [comment]
> > > > > > ・不加分问题是，我不能让它工作，所以你可以只是建立当前阶段，并将其作为某种预发布版本？会很好的
> > > > > 
> > > > > 
> > > > > 大人种https://github.com/joogle12/SeedCracker/releases/tag/1.17-release
> > > > 
> > > > 
> > > > 事实上，您创建的种子爆竹可能仅适用于原始 1.17 版本。如果服务器添加洞穴和悬崖包，它可能无法正常工作。我不知道你是否可以更新一个版本的洞穴和悬崖包？
> > > 
> > > 
> > > 洞穴和悬崖不是在1.18年出来的吗？地下生物群系等？
> > 
> > 
> > Mojang 打开洞穴和悬崖包，让玩家提前体验 1.18 内容。您可以在维基上搜索更多信息。它被称为洞穴和悬崖预览数据包。此数据包已添加到我正在播放的服务器。我和我的朋友无法使用您的工具确定种子，所以我们怀疑这是否是因为服务器添加了此数据包。
> 
> 嘿。对不起，我不知道该怎么做。我只从源头构建了mod并将其共享。

Ok

**Comment by joogle12** [comment]
> > > > > 嘿，伙计，我知道它还没有完成1.17，但它似乎会工作1.17时，建立它与gradle。问题是，我不能让它工作，所以你可以只是建立当前阶段，并将其作为某种预发布版本？会很好的
> > > > 
> > > > 
> > > > 我做了一个https://github.com/joogle12/SeedCracker/releases/tag/1.17-release
> > > 
> > > 
> > > 事实上，您创建的种子爆竹可能仅适用于原始 1.17 版本。如果服务器添加洞穴和悬崖包，它可能无法正常工作。我不知道你是否可以更新一个版本的洞穴和悬崖包？
> > 
> > 
> > 洞穴和悬崖不是在1.18年出来的吗？地下生物群系等？
> 
> Mojang opens up caves and cliff packets for players to experience 1.18 content in advance. You can search the wiki for more information. It's named Cave & Cliffs Preview Datapack. This datapack is added to the server I am playing. My friends and I were unable to determine the seed using your tool, so we suspected whether it was because the server added this packet.

hey man. I published a pre-release for the thing you were asking for. Try it out and let me know. 

https://github.com/joogle12/SeedCracker/releases/tag/1.18Preview
